docs/user/user/adding-glean-to-your-project/enable-glean-dictionary.md

```diff
@@ -0,0 +1,31 @@
+# Adding your product to the Glean Dictionary
+
+This page provides a step-by-step guide on how to add your project to the [Glean Dictionary](https://dictionary.telemetry.mozilla.org/).
+
+This is just one of the required steps for integrating Glean successfully into a project. Check the full [Glean integration checklist](./index.md) for a comprehensive list of all the steps involved in doing so.
+
+# Requirements
+
+GitHub Actions
+
+# Enable your product
+
+At least one week before releasing your product, [file a data engineering bug][dataeng-bug] to enable your product's application id and metrics to be published to the Glean Dictionary. This will result in your product being added to `probe-scraper`'s [`repositories.yaml`](https://github.com/mozilla/probe-scraper/blob/main/repositories.yaml)
+
+# Validate and publish metrics
+
+After your product has been enabled, you must submit commits to [`probe-scraper`](https://github.com/mozilla/probe-scraper) to validate and publish metrics to the Glean Dictionary. Metrics will only be published from branches defined in [`repositories.yaml`](https://github.com/mozilla/probe-scraper/blob/main/repositories.yaml), or the git default branch if not explicitly configured.
+
+Add the following `.github/workflows/glean-probe-scraper.yml` to submit commits via GitHub workflows:
+
+```yaml
+---
+name: GLEAN probe-scraper
+on: [push, pull_request]
+jobs:
+  glean-probe-scraper:
+    steps:
+    - uses: mozilla/probe-scraper/.github/workflows/glean.yaml@main
```

Comment:
> Oh, does it work that way? I only knew the `org/repo@tag` syntax.

org/repo@tag is for actions, this is a "re-usable workflow". The important difference is that in a re-usable workflow I can specify `runs-on: ubuntu-latest`, so that it isn't specified here and is under our control.

> Also: do we need to allow-list the action with GitHub admins?

I was able to use `mozilla/probe-scraper/.github/workflows/glean.yaml@probe-scraper-push` from a personal repo, so I don't think so. 

> Does this task run synchronously and can communicate back failure? If not what is the way for users to understand whether everything went ok (except them asking us later)?

yes, it runs synchronously and reports back failure messages.

Reply:
Thanks! That clarifies it for me. Wasn't aware of reusable workflows like that.